src/v0.4/parseExpr_staged.jl

```diff
@@ -383,10 +383,10 @@ function parseNorm(x::Expr, aff::Symbol, lcoeffs, rcoeffs, newaff)
                 $code
             end)
         end
-        preblock = :($normexpr = AffExpr[])
+        preblock = :($normexpr = GenericAffExpr[])
```

Comment:
I'm not sure this'll work when you try to construct the ``Norm``, and I wonder if there will be performance hits.